pkg/scheduler/api/job_info.go

```diff
@@ -19,18 +19,30 @@ package api
 import (
 	"fmt"
 	"sort"
-	"strconv"
 	"strings"
 
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
-	"k8s.io/klog"
 
 	"volcano.sh/volcano/pkg/apis/scheduling"
 	"volcano.sh/volcano/pkg/apis/scheduling/v1beta1"
 )
 
+// PreemptType has preemptable, non-preemptable, revocable types
+type PreemptType int
+
+const (
+	// Preemptable means the task can be scheduled to revocable nodes and can be evict and preempted.
+	Preemptable PreemptType = 1 << iota
+
+	// NonPreemptable means the task can not be scheduled to revocable and can preempt Preemptable tasks
+	NonPreemptable
+
+	// Revocable means the task can be scheduled to revocable nodes but can not be evict or preempted
+	Revocable
```

Comment:
Finally understand your intention, however it is a bit confusing. One words contains multiple meanings actually and from the word itself only indicts one meaning.